packages/dd-trace/src/appsec/index.js

```diff
@@ -72,10 +73,33 @@ function incomingHttpEndTranslator (data) {
   const headers = Object.assign({}, data.res.getHeaders())
   delete headers['set-cookie']
 
-  Gateway.propagate({
+  const payload = {
     [addresses.HTTP_INCOMING_RESPONSE_CODE]: data.res.statusCode,
     [addresses.HTTP_INCOMING_RESPONSE_HEADERS]: headers
-  }, context)
+  }
+
+  // TODO: temporary express instrumentation, will use express plugin later
+  if (data.req.body !== undefined && data.req.body !== null) {
```

Comment:
Let's not do the body in this iteration to ensure good sleep for all of us (the binding's object limitation do not cover the size of objects, only their depth)

Reply:
Done